bin/test-msg-prod.sh

```diff
@@ -0,0 +1,38 @@
+"records": [
+	{
+		"value": {
+			"private_efdStamp": 1674516757.7661333,
+			"private_kafkaStamp": 1674516794.7740011,
+			"salIndex": 2,
+			"private_revCode": "c9aab3df",
+			"private_sndStamp": 1674516794.7661333,
+			"private_rcvStamp": 1674516794.7662647,
+			"private_seqNum": 22,
+			"private_identity": "ScriptQueue:2",
+			"private_origin": 2786567,
+			"scriptSalIndex": 200023,
+			"groupId": "Test",
+			"coordinateSystem": 1,
+			"position": [
+				0,
+				0
+			],
+			"rotationSystem": 1,
```

Comment:
No `startTime` or `instrument`?

Reply:
Thanks for catching this!  I did not test-run `bin/test-msg-prod.sh` but simply moved the same file here. I don't think I have the permissions to post on `prod`'s nextVisit, and I probably just should not. 

Thinking it more, I probably won't move this file here.  The only meaningful difference from `test-msg-dev.sh` is the sasquatch url.

Reply:
No longer including this script in this PR.